Lib/bdb.py

```diff
@@ -62,12 +104,28 @@ def trace_dispatch(self, frame, event, arg):
         return self.trace_dispatch
 
     def dispatch_line(self, frame):
+        """Trace function when a new line of code is executed.
```

Comment:
Hmm.  Just saying "Return trace function." does not really help much.  It is really an implementation detail in that communicating a tracer change could have been differently. setting self.tracer, for instance, could have been the linking mechanism.  Describing the 'side-effect', which is actually the main effect is awkward. "Return trace function after calling user_line if debugger stops here." is fairly accurate and descriptive.  But I am open to other ideas for the wording.  Ditto for other 3 event handlers.